src/components/TextInput.tsx

```diff
@@ -75,9 +75,9 @@ export class CTextInput extends React.Component<Props, State> {
         <RNTextInput
           ref={forwardedRef}
           style={[
+            inputStyle ?? fontStyles.regular,
```

Comment:
This is still a bit of a head-scratcher for me. Basically, this was never respecting font styling overrides. If `fontStyles.regular` was included _anywhere_ (either above or below `inputStyle`) the font styling in `inputStyle` would never be respected. This fix required a handful of snapshots to be updated. An alternative solution could have been to add a new param to this component such that existing snapshots wouldn't need to be updated, but that felt pretty hacky, considering we already have at least two styling props on the component.

Reply:
might make sense to put this in its own PR

Reply:
if styles.input comes later than inputStyle, wouldn't this mean that padding stuff can never be overridden? It seems like some of the usages of the inputStyle is overriding the padding. A few examples:
- https://github.com/valora-inc/wallet/blob/1b382a974557a7e51aaf21b91073613bb31740cf/src/swap/SwapAmountInput.tsx#L82
- https://github.com/valora-inc/wallet/blob/1b382a974557a7e51aaf21b91073613bb31740cf/src/components/SearchInput.tsx#L17

Also, +1 to making these changes on a separate PR